h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -194,6 +194,10 @@ def _train(self, parms, verbose=False):
         model_builder_json = h2o.api("POST /%d/ModelBuilders/%s" % (rest_ver, self.algo), data=parms)
         job = H2OJob(model_builder_json, job_type=(self.algo + " Model Build"))
 
+        for mesg in model_builder_json["messages"]:
+            if mesg["message_type"] == "WARN":
+                warnings.warn(mesg["message"], RuntimeWarning)
+
```

Comment:
according to `ModelBuilderSchema`, looks like there is a possibility for `messages` to be null/None.
Maybe it doesn't occur in practice though... doesn't the R client do any check?

Reply:
Thank you, that's a good point. R checks length... I ran locally all python stacked ensemble tests and it didn't fail on anything - possibly due to always having some warning... But better safe than sorry...